harper-core/src/linting/else_possessive.rs

```diff
@@ -0,0 +1,96 @@
+use crate::{
+    Token,
+    linting::{Lint, LintKind, PatternLinter, Suggestion},
+    patterns::{Pattern, SequencePattern, WordSet},
+};
+
+pub struct ElsePossessive {
+    pattern: Box<dyn Pattern>,
+}
+
+impl Default for ElsePossessive {
+    fn default() -> Self {
+        let pronouns = WordSet::new(&[
+            "somebody",
+            "someone",
+            "anybody",
+            "anyone",
+            "everybody",
+            "everyone",
+            "nobody",
+            "noone",
```

Comment:
"no one" is <b>also</b> legit. In fact I just checked and "noone" is actually a mistake. You should be able to use an `EitherPattern` with the `WordSet` on one side and `SequencePattern` of `no` and `one` on the other.